MonoGame.Framework/Graphics/Effect/IEffectFog.cs

```diff
@@ -48,32 +10,31 @@ namespace Microsoft.Xna.Framework.Graphics
 	public interface IEffectFog
 	{
         /// <summary>
-        /// The floating point fog color.
+        /// Gets or set the fog color in floating point format.

```

Comment:
That's fine.  The extra "gets or sets" will be redundant because the vast majority of properties are read/write.